framework/src/controller/controller.js

```diff
@@ -203,23 +199,38 @@ class Controller {
 			`Loading module with alias: ${moduleAlias}(${name}:${version})`
 		);
 
-		const channel = new ChildProcessChannel(
-			moduleAlias,
-			module.events,
-			module.actions
+		const modulePath = path.resolve(
+			__dirname,
+			'../modules',
+			alias.replace(/([A-Z])/g, $1 => `_${$1.toLowerCase()}`)
 		);
 
-		await channel.registerToBus(this.socketsPath);
+		const program = path.resolve(__dirname, 'child_process_loader.js');
 
-		channel.publish(`${moduleAlias}:registeredToBus`);
-		channel.publish(`${moduleAlias}:loading:started`);
-		await module.load(channel);
-		channel.publish(`${moduleAlias}:loading:finished`);
+		const parameters = [
+			modulePath,
+			JSON.stringify({ ...options, socketsPath: this.socketsPath }),
+		];
+
+		const child = child_process.fork(program, parameters);
+
+		child.on('exit', () => {
+			this.logger.error(`Module ${moduleAlias}(${name}:${version}) exited`);
```

Comment:
It would be nice here to have the child have an error handler that sends the error that cause the exit back to its parent so we have more information on the exit reason.